src/signals/incident-management/index.js

```diff
@@ -53,24 +53,38 @@ const AreaContainer = lazy(() => import('./containers/AreaContainer'))
 const SignalingContainer = lazy(() => import('./containers/SignalingContainer'))
 
 const IncidentManagement = () => {
-  // const location = useLocationReferrer()
   const districts = useSelector(makeSelectDistricts)
-  const searchQuery = useSelector(makeSelectSearchQuery)
+  const searchQueryIncidents = useSelector(makeSelectSearchQuery)
   const dispatch = useDispatch()
+  const [page, setPage] = useState(1)
```

Comment:
a bit strange when a page is not starting from 0

Reply:
i mean normally in code i would always start it from 0 and then + 1 it in the ui. (jsx)